docs/proto-docs.md

```diff
@@ -5636,8 +5636,9 @@ MsgAssessCustomMsgFeeRequest defines an sdk.Msg type
 | ----- | ---- | ----- | ----------- |
 | `name` | [string](#string) |  | optional short name for custom msg fee, this will be emitted as a property of the event |
 | `amount` | [cosmos.base.v1beta1.Coin](#cosmos.base.v1beta1.Coin) |  | amount of additional fee that must be paid |
-| `recipient` | [string](#string) |  | optional recipient address, the total amount is given to the recipient, if present. |
+| `recipient` | [string](#string) |  | optional recipient address, the basis points amount is sent to the recipient |
 | `from` | [string](#string) |  | the signer of the msg |
+| `recipient_basis_points` | [string](#string) |  | optional basis points 0 - 10,000 for recipient defaults to 10,000 |
```

Comment:
Yes, this allows for use with our default value.  It it isn't set we use the internal default value for assess msg bips.  This also allows for backwards compatibility with contracts that already use the msg.  They will not have to change anything to their contracts to keep the same behavior.  If it was set to a uint or int it would have defaulted to 0 when the proto is used.  I might be missing a better impl here?